README.md

```diff
@@ -5,51 +5,50 @@ Login by Auth0
 
-[Please see the Installation page on auth0.com/docs](https://auth0.com/docs/cms/wordpress/installation)
-
-## Extending the plugin
-
-[Please see the Extending page on auth0.com/docs](https://auth0.com/docs/cms/wordpress/extending)
-
-We're happy to review and approve new filters and actions that help you integrate even further in this plugin. Please
- see the Contributing section at the bottom of this page.
+Please see the [Installation page on auth0.com/docs](https://auth0.com/docs/cms/wordpress/installation) for detailed 
+instructions.
 
 ## API authentication
 
-This plugin provides the ability integrate with **wp-jwt-auth** plugin to authenticate api calls via a HTTP Authorization Header.
+Please see the [JWT Authentication page on auth0.com/docs](https://auth0.com/docs/cms/wordpress/jwt-authentication) 
+for more information. 
 
-This plugin will detect if you have wp-jwt-auth installed and active and will offer to configure it. Accepting this, will set up the client id, secret and the custom user repository.
+## Extending the plugin
 
-After the user logs in via Auth0 in your Api client (ie: using Lock in a mobile app) you will get a JWT (JSON Web Token). Then you need to send this token in a HTTP header in the following way:
+Please see the [Extending page on auth0.com/docs](https://auth0.com/docs/cms/wordpress/extending) for documentation on existing hooks.
 
-```
-Authorization: Bearer ##jwt##
-```
+We're happy to review and approve new filters and actions that help you integrate even further in this plugin. Please
+ see the **Contributing** section below for more information.
 
-For example:
+## Contributing
 
-```
-Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJjb250ZW50IjoiVGhpcyBpcyB5b3VyIHVzZXIgSldUIHByb3ZpZGVkIGJ5IHRoZSBBdXRoMCBzZXJ2ZXIifQ.b47GoWoY_5n4jIyG0hPTLFEQtSegnVydcvl6gpWNeUE
-```
+**Thank you in advance!**
 
-This JWT should match with a registered user in your WP installation.
+* All PRs must use the `dev` branch as a base
+* Change the least amount of code to achieve the goal; PRs with lots of whitespace changes and abstraction combined 
+with a feature add or bug fix are difficult to review and may be rejected
+* Use the latest version of WordPress and turn `WP_DEBUG` on
+* If other plugins are installed during testing that might affect behavior, please list those in the PR description
+* Make sure to test against the lowest PHP version supported (see `Requires PHP:` [here](https://github.com/auth0/wp-auth0/blob/master/readme.txt#L5)) 
 
-You can use this feature with API's provided by plugins like **WP REST API (WP API)**.
+### How to install and configure WordPress for testing
 
-## Contributing
+We try to cover as many use cases as possible and one way to do that is with a "headless" WordPress install (core WordPress files in a separate directory). Here's a quick and easy way to set that up on the command line:
 
-Thank you in advance!
+1. `mkdir wp-doc-root; cd wp-doc-root; mkdir wp;`
```

Comment:
might want to put them all inside a code block to highlight the syntax

````bash
mkdir wp-doc-root; cd wp-doc-root; mkdir wp;
mkdir wp-content; mkdir wp-content/themes; mkdir wp-content/plugins;
git clone https://github.com/WordPress/WordPress.git wp; # takes a while
cp ./wp/wp-config-sample.php ./wp-config.php; # edit to add MySQL DB info and set "WP_DEBUG" to TRUE

# You can even have one comment per line to be more clear about what the command is doing

cp ./wp/index.php ./index.php; # replace "/wp-blog-header.php" with "/wp/wp-blog-header.php"
git clone https://github.com/joshcanhelp/auth0-wp-test.git ./wp-content/themes/auth0-wp-test
````

Reply:
Just had this as a list for order-of-ops but I like the block better